templates/app/app.txt

```diff
@@ -47,7 +47,7 @@ requireAll({
 requireAll({
   dirname: path.join(__dirname, 'middlewares'),
   recursive: true,
-  resolve: Module => new Module(app),
```

Comment:
<img width="668" alt="Screenshot 2019-12-02 at 17 40 00" src="https://user-images.githubusercontent.com/3200482/69977255-ca982600-152a-11ea-92de-b0c113844696.png">
We remove the new here because an sync function can't be used as a constructor. (Furthermore 'Module' is not a class but a Function)